bugsnag/src/main/java/com/bugsnag/callbacks/ServletCallback.java

```diff
@@ -36,7 +36,8 @@ public void beforeNotify(Report report) {
         report
                 .addToTab("request", "url", request.getRequestURL().toString())
                 .addToTab("request", "method", request.getMethod())
-                .addToTab("request", "params", request.getParameterMap())
+                .addToTab("request", "params",
+                        new HashMap<String, String[]>(request.getParameterMap()))
```

Comment:
This was required for testing because when the request context has ended, the parameter map is no longer available, even though the report object might still exist. Copying the context into a new list before this happens keeps the values available for the lifetime of the report object.